content/dart/concept/list/terms/remove/remove.md

```diff
@@ -0,0 +1,56 @@
+
+
+---
+Title: 'List.remove()'
+Description: 'Removes first occurrence of a value from the list.'
+Subjects:
+  - 'Andriod Development'
+  - 'Dart'
+Tags:
+  - 'List'
+  - 'Remove'
+  - 'Method'
```

Comment:
Choose the tags from this [page](https://github.com/Codecademy/docs/blob/main/documentation/tags.md).